src/GovUk.Education.ExploreEducationStatistics.Admin/Requests/ReleaseFileRequests.cs

```diff
@@ -0,0 +1,35 @@
+#nullable enable
+using System.ComponentModel.DataAnnotations;
+using Microsoft.AspNetCore.Http;
+
+namespace GovUk.Education.ExploreEducationStatistics.Admin.Requests;
+
+public record ReleaseFileUpdateRequest
+{
+    public string? Title { get; set; }
+
+    public string? Summary { get; set; }
+}
+
+public record ReleaseAncillaryFileUploadRequest
+{
+    [Required]
+    public string Title { get; set; } = null!;
+
+    [Required]
+    public string Summary { get; set; } = null!;
```

Comment:
Would suggest initialising these as empty strings to be consistent with other places.

Same below for the update request.